src/parser.js

```diff
@@ -2081,7 +2081,7 @@ export class GenericParser extends Tokenizer {
     if (!isGenerator && token.type === TokenType.IDENTIFIER) {
       if (token.value.length === 3) {
         // Property Assignment: Getter and Setter.
-        if (token.value === 'get' && this.lookaheadPropertyName()) {
+        if (!token.escaped && token.value === 'get' && this.lookaheadPropertyName()) {
```

Comment:
Check the things that are more likely to fail earlier. In these cases, `token.value === 'get'` is more likely to be `false` than `!token.escaped`.